tests/large-atoms/input.dat

```diff
@@ -1,124 +1,56 @@
-#! Sample with post-Argon atoms !nosample
+#! Test SCF and DF-MP2 on post-Ar atoms
 
 molecule {
 Ba
-Ba 1 2.0
-units bohr
+Ba 1 4.0
```

Comment:
Might the approximate unit conversion have gone the wrong way?

Reply:
Ah, no. At 2 bohr, the gradient is huge (for Ba-Ba, ~15 Hartree/bohr) which makes getting converged finite differences really tough. I assumed these inputs were boilerplate and made the geometries more sensible (unless having overlapping atoms here was intentional?).

Reply:
yes, they're boilerplate, so sure, whatever's sensible. using Bohr can be a little easier to match between programs.